### DISCUSSION TITLE
issue with apk

### DISCUSSION THREAD

**Discussion by NITHISH-KUMAR-U** [erfansn/AR-Touch#34] [Q&A]
when we install android application in our phone it crashes.not working.what is the solution why it happen

**Comment by erfansn** [comment]
Thank you for reporting the problem, but it would have been better if you had raised it in the [issue](https://github.com/ErfanSn/AR-Touch/issues/) section. Anyway, please report the Android version and model of the tested phone as a [new issue](https://github.com/ErfanSn/AR-Touch/issues/new) so that I can better solve the problem.
